bridge-token/src/lib.rs

```diff
@@ -30,13 +30,15 @@ pub struct BridgeToken {
     decimals: u8,
     paused: Mask,
     icon: Option<String>,
+    new_controller: Option<AccountId>
```

Comment:
No needs to complicate the token itself, the controller can be staged on the factory side.

Reply:
Fixed in https://github.com/Near-One/rainbow-token-connector/pull/253